modules/SantoriniBoard.class.php

```diff
@@ -57,45 +61,70 @@ public static function addInfo($piece)
     } elseif (substr($piece['type'], 0, 5) == 'token') {
       $piece['direction'] = $piece['type_arg'];
     }
+    Utils::convertIntValues($piece);
```

Comment:
When we load pieces from the database, we convert the fields like x,y,z,id,etc. from string to int right away, then we do not need (int) casting later.

Reply:
Shouldn't it be      Utils::convertIntValues($data); instead of      Utils::convertIntValues($mixed); on the getCoords function ?